Masonry/MASViewAttribute.h

```diff
@@ -14,6 +14,8 @@
  */
 @interface MASViewAttribute : NSObject
 
+NS_ASSUME_NONNULL_BEGIN
+
 /**
  *  The view which the reciever relates to. Can be nil if item is not a view.
  */
```

Comment:
actually now _did_ fix the convenience initializer, as the `item` property is `weak` and thus needs to be `nullable`